### ISSUE TITLE
Pause method not working. isPlaying alway return false

### ISSUE THREAD

**Issue by trungdiep92** [wseemann/FFmpegMediaPlayer#121]
Hi! I call pause method of FfmpegMediaPlayer. But it not working and  isPlaying() alway return false.
Create media player code:
` try {
            if (mMediaPlayer != null) {
                mMediaPlayer.release();
            }
            mMediaPlayer = new FFmpegMediaPlayer();
            mMediaPlayer.setOnErrorListener(mOnErrorListener);
            mMediaPlayer.stop();
            mMediaPlayer.setOnCompletionListener(mOnCompletionListener);
            mMediaPlayer.reset();
            mMediaPlayer.setDataSource(urlToStream);
            mmSurfaceView.setOnTouchListener(this);

            if (mFinalSurface != null) {
                mMediaPlayer.setSurface(mFinalSurface);
            }
            mMediaPlayer.setOnPreparedListener(mOnPreparedListener);
            mMediaPlayer.prepareAsync();
        } catch (IOException ex) {
            ex.printStackTrace();
        }`
Pause and start:
` if (playing) {
                        boolean isP = mMediaPlayer.isPlaying(); //alway return false
                        mMediaPlayer.stop();
                        btnPlayPause.setImageResource(R.drawable.ic_action_play_over_video);
                        playing = false;
                    } else {
                        mMediaPlayer.start();
                        btnPlayPause.setImageResource(R.drawable.ic_action_pause_over_video);
                        playing = true;
                    }`

Please help me. Thanks!

**Comment by Farrukh381** [comment]
Yes Pause method does not work .....Pause Method Just make the volume Mute
Please Help Thanks

**Comment by qinkaituhu** [comment]
Did this problem solvied?

**Comment by Farrukh381** [comment]
No

**Comment by qinkaituhu** [comment]
@Farrukh381 
https://github.com/Bilibili/ijkplayer 
这个不错，推荐下
